backend/tests/api/test_history.py

```diff
@@ -0,0 +1,148 @@
+from uuid import uuid4
+
+import pytest
+from app.auth.config import API_SECRET_KEY
+from app.history.routers import router
+from fastapi import HTTPException
+from fastapi.testclient import TestClient
+
+client = TestClient(router)
+
+
+@pytest.mark.parametrize(
+    "history_data, expected_status",
+    [
+        (
+            {
+                "chat_id": str(uuid4()),
+                "created_by": "Alice",
+                "responses": [
+                    {
+                        "question": "test question",
+                        "answer": "This is the first response.",
+                    }
+                ],
+            },
+            200,
+        ),
+        (
+            {
+                "chat_id": str(uuid4()),
+                "created_by": "Bob",
+                "responses": [],
+            },
+            200,
+        ),
+    ],
+)
+def test_user_history(history_data: dict, expected_status: int) -> None:
+    headers = {
+        "accept": "application/json",
+        "Authorization": f"Bearer {API_SECRET_KEY}",
+    }
+    response = client.post("/history", json=history_data, headers=headers)
+    assert (
+        response.status_code == expected_status
+    )  # Ensure the status code matches the expected one
```

Comment:
Suggest removing comments to make it less cluttered. Code seems pretty self-explanatory

Reply:
I prefer the comment like this